test/performance/broker-imc/continuous/100-broker-imc-setup.yaml

```diff
@@ -0,0 +1,182 @@
+# Copyright 2019 The Knative Authors
```

Comment:
Is this setup file the same as the existing one under `broker-imc`?

Reply:
same here, @grantr why did you duplicated configs?